### PR TITLE
switched from comboHit::time to comboHit::correctedTime in Cpr

### PR DESCRIPTION
**Pull request by gianipez**

### CODE REVIEW

**Comment by gianipez** [comment]
tagging @brownd1978 @pavel1murat 
I'm checking the final rates and post some slides on docdb soon.

Cheers,

Giani

File: CalPatRec/src/CalHelixFinder_module.cc [hunk 1]
```diff
@@ -338,6 +338,9 @@ namespace mu2e {
 
 	int loc = _data.nseeds[0];
 	if (loc < _data.maxSeeds()) {
+	  if (index_best == 2){
```
Comment (brownd1978) [inline CalPatRec/src/CalHelixFinder_module.cc:341]: I don't understand this logic.  What does index_best==2 correspond to physically?

File: CalPatRec/src/CalTimePeakFinder_module.cc [hunk 2]
```diff
@@ -200,7 +200,7 @@ namespace mu2e {
     static const double twopi(2*pi);
 
     double              mphi(-9999.);
-    double              meanDriftTime = 1.25/0.06;// half straw tube radius / drift velocity
+    double              meanDriftTime = -3.3;//1.25/0.06;// half straw tube radius / drift velocity
```
Comment (brownd1978) [inline CalPatRec/src/CalTimePeakFinder_module.cc:203]: What does this number correspond to?  If it depends on gas properties it should come from StrawResponse.

File: CalPatRec/src/CalTimePeakFinder_module.cc [hunk 3]
```diff
@@ -243,7 +243,7 @@ namespace mu2e {
           for(int istr=0; istr<nch;++istr) {
 
             hit    = &_data.chcol->at(istr);
-            time   = hit->time();
+            time   = hit->correctedTime();//time();
```
Comment (brownd1978) [inline CalPatRec/src/CalTimePeakFinder_module.cc:246]: Please remove replaced code

**Review by brownd1978 — COMMENTED** [review]
Can you explain how the DtMin,Max were tuned?  I guess that will be in your talk tomorrow.

File: CalPatRec/src/CalTimePeakFinder_module.cc (hunk 2 above)
Comment (gianipez) [reply CalPatRec/src/CalTimePeakFinder_module.cc:203]: Waiting for @bonventre to comment on this

File: CalPatRec/src/CalTimePeakFinder_module.cc (hunk 3 above)
Comment (gianipez) [reply CalPatRec/src/CalTimePeakFinder_module.cc]: done

File: CalPatRec/src/CalHelixFinder_module.cc (hunk 1 above)
Comment (gianipez) [reply CalPatRec/src/CalHelixFinder_module.cc:339]: the case where two helices will be stored. The current diagnostic doesn't handle such a case, that's why I'm selecting the first one. This is OK as I usually run the diagnostic only with the single-particle datasets

File: CalPatRec/src/CalTimePeakFinder_module.cc (hunk 2 above)
Comment (bonventre) [reply CalPatRec/src/CalTimePeakFinder_module.cc]: The TOT calibration currently has a 3 ns bias (needs to be recalibrated based on MDC2020 samples), so this is expected right now but should be closer to zero once that's fixed. After recalibrating should check this against https://github.com/Mu2e/Offline/blob/main/TrkReco/inc/TrkTimeCalculator.hh#L37, maybe should be the same fcl parameter if there still ends up being an offset

Comment (gianipez) [reply CalPatRec/src/CalTimePeakFinder_module.cc]: good point. I use now an fcl parameter that was already there.

**Comment by gianipez** [comment]
Hi there,
I got rid of the hardcoded value and instead incorporated this new offset in the existing one.

Please, let me know if further actions are needed.

Cheers,

Giani

**Comment by gianipez** [comment]
@FNALbuild run build test

**Comment by gianipez** [comment]
@FNALbuild run build test

**Review by brownd1978 — APPROVED** [review]
I note that there are a number of hard-coded time offsets, these need to be addressed in a more systematic way, though that's outside the scope of this PR

**Review by bonventre — APPROVED** [review]

**Pull request merged**